---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

<!--Author profiles on <a href="https://scholar.google.com/citations?user=eAPnyCsAAAAJ&hl" target="_blank">Google Scholar</a>.-->

## Preprints ##
* **Importance is Important: A Guide to Informed Importance Tempering Methods.** <br/>
*Li, G.*, Smith, A. and Zhou, Q., 2023. arXiv preprint arXiv:2304.06251. <a href="https://arxiv.org/pdf/2304.06251.pdf" target="_blank">(arXiv)</a>

* **Interpretable modeling of time-resolved single-cell gene-protein expression using CrossmodalNet.** <br/>
Yang, Y., Lin, Y.T., *Li, G.*, Zhong, Y., Xu, Q. and Cai, J.J., 2023.  bioRxiv, pp.2023-05. <a href="https://www.biorxiv.org/content/10.1101/2023.05.16.541011v2" target="_blank">(bioRxiv)</a>

* **Fast Replica Exchange Stochastic Gradient Langevin Dynamics.** <br/>
*Li, G.*, Lin, G., Zhang, Z. and Zhou, Q., 2023. arXiv preprint arXiv:2301.01898. <a href="https://arxiv.org/abs/2301.01898" target="_blank">(arXiv)</a>

## Journal Publications ##
* **Bayesian Multi-task Variable Selection with an Application to Differential DAG Analysis.** <br/>
*Li. G.* and Zhou. Q., 2023. Accepted by Journal of Computational and Graphical Statistics. <a href="https://arxiv.org/pdf/2206.14303.pdf" target="_blank">(arXiv)</a>

* **Gene knockout inference with variational graph autoencoder learning single-cell gene regulatory networks.** <br/>
Yang, Y., *Li, G.*, Zhong, Y., Xu, Q., Chen, B.J., Lin, Y.T., Chapkin, R.S. and Cai, J.J., 2023. Nucleic Acids Research, p.gkad450. <a href="https://academic.oup.com/nar/article/51/13/6578/7184155" target="_blank">(link)</a>

* **scTenifoldXct: A semi-supervised method for predicting cell-cell interactions and mapping cellular communication graphs.** <br/>
Yang, Y., *Li, G.*, Zhong, Y., Xu, Q., Lin, Y.T., Roman-Vicharra, C., Chapkin, R.S. and Cai, J.J., 2023. Cell Systems, 14(4), pp.302-311. <a href="https://www.cell.com/cell-systems/pdf/S2405-4712(23)00030-3.pdf" target="_blank">(link)</a>

* **scTenifoldKnk: An efficient virtual knockout tool for gene function predictions via single-cell gene regulatory network perturbation.** <br/>
Osorio, D., Zhong, Y., *Li, G.*, Xu, Q., Yang, Y., Tian, Y., Chapkin, R.S., Huang, J.Z. and Cai, J.J., 2022. Patterns, 3(3), p.100434. <a href="https://pubmed.ncbi.nlm.nih.gov/35510185/" target="_blank">(link)</a>

* **scInTime: A Computational Method Leveraging Single-Cell Trajectory and Gene Regulatory Networks to Identify Master Regulators of Cellular Differentiation.** <br/>
Xu, Q., *Li, G.*, Osorio, D., Zhong, Y., Yang, Y., Lin, Y.T., Zhang, X. and Cai, J.J., 2022. Genes, 13(2), p.371. <a href="https://www.mdpi.com/2073-4425/13/2/371" target="_blank">(link)</a>

* **scTenifoldNet: a machine learning workflow for constructing and comparing transcriptome-wide gene regulatory networks from single-cell data.** <br/>
Osorio, D., Zhong, Y., *Li, G.*, Huang, J.Z. and Cai, J.J., 2020. Patterns, 1(9), p.100139. <a href="https://www.sciencedirect.com/science/article/pii/S2666389920301872" target="_blank">(link)</a>

* **Single-cell expression variability implies cell function.** <br/>
Osorio, D., Yu, X., Zhong, Y., \textbf{Li, G.}, Serpedin, E., Huang, J.Z. and Cai, J.J., 2019.  Cells, 9(1), p.14 <a href="https://pubmed.ncbi.nlm.nih.gov/31861624/" target="_blank">(link)</a>
