---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

You can view my author profiles on [Google Scholar](https://scholar.google.com/citations?user=jxFkCp8AAAAJ&hl=en).

_Notation_: <sup>♯</sup> corresponding author; <sup>*</sup> co-first author; <sup>†</sup> student supervised by me

### Preprints
- **Li, G.** and Zhang, X. (2025). A General Framework for Multiple Testing via E-value Aggregation and Data-Dependent Weighting. [(arXiv)](https://arxiv.org/abs/2312.02905).
- **Li, G.**, Smith, A., and Zhou, Q. (2024). Importance is Important: Generalized Markov Chain Importance Sampling Methods. [(arXiv)](https://arxiv.org/abs/2304.06251).
- **Li, G.**, Lin, G., Zhang, Z., and Zhou, Q. (2023). Fast Replica Exchange Stochastic Gradient Langevin Dynamics. [(arXiv)](https://arxiv.org/abs/2301.01898).

### Journal Publications
- Gao, J.<sup>†</sup>, Chen, H., and **Li, G.<sup>♯</sup>** (2025). Data-Dependent Weighted E-value Aggregation for Fusion Learning. _Mathematics_.
- Li, X., Liu, X.<sup>*,†</sup>, and **Li, G.<sup>♯</sup>** (2025). Adaptive Testing for Segmenting Watermarked Texts From Language Models. _Stat_, 14(4).
- Zhong, Y., Osorio, D., **Li, G.**, Xu, Q., Yang, Y., Huang, J. Z., and Cai, J. J. (2025). scTenifoldNet and scTenifoldKnk: A package suite for single-cell gene regulatory network construction, comparison, and perturbation analysis. _Statistical Theory and Related Fields_, pp. 1–10.
- **Li, G.** and Zhang, X. (2025). A Note on E-values and Multiple Testing. _Biometrika_, 112(1).
- **Li, G.** and Zhou, Q. (2024). Bayesian Multi-Task Variable Selection with an Application to Differential DAG Analysis. _Journal of Computational and Graphical Statistics_, 33(1), pp. 35–46.
- **Li, G.**, Lu, Y., Chen, J., and Zhang, X. (2023). Robust Differential Abundance Analysis of Microbiome Sequencing Data. _Genes_, 14(11), p. 2000.
- Yang, Y., Lin, Y. T., **Li, G.**, Zhong, Y., Xu, Q., and Cai, J. J. (2023). Interpretable modeling of time-resolved single-cell gene–protein expression with CrossmodalNet. _Briefings in Bioinformatics_, 24(6), p. bbad342.
- Yang, Y., **Li, G.**, Zhong, Y., Xu, Q., Chen, B. J., Lin, Y. T., Chapkin, R. S., and Cai, J. J. (2023). Gene knockout inference with variational graph autoencoder learning single-cell gene regulatory networks. _Nucleic Acids Research_, p. gkad450.
- Yang, Y., **Li, G.**, Zhong, Y., Xu, Q., Lin, Y. T., Roman-Vicharra, C., Chapkin, R. S., and Cai, J. J. (2023). scTenifoldXct: A semi-supervised method for predicting cell–cell interactions and mapping cellular communication graphs. _Cell Systems_, 14(4), pp. 302–311.
- Osorio, D., Zhong, Y., **Li, G.**, Xu, Q., Yang, Y., Tian, Y., Chapkin, R. S., Huang, J. Z., and Cai, J. J. (2022). scTenifoldKnk: An efficient virtual knockout tool for gene function predictions via single-cell gene regulatory network perturbation. _Patterns_, 3(3), p. 100434.
- Xu, Q., **Li, G.**, Osorio, D., Zhong, Y., Yang, Y., Lin, Y. T., Zhang, X., and Cai, J. J. (2022). scInTime: A computational method leveraging single-cell trajectory and gene regulatory networks to identify master regulators of cellular differentiation. _Genes_, 13(2), p. 371.
- Osorio, D., Zhong, Y., **Li, G.**, Huang, J. Z., and Cai, J. J. (2020). scTenifoldNet: A machine learning workflow for constructing and comparing transcriptome-wide gene regulatory networks from single-cell data. _Patterns_, 1(9), p. 100139.
- Osorio, D., Yu, X., Zhong, Y., **Li, G.**, Serpedin, E., Huang, J. Z., and Cai, J. J. (2019). Single-cell expression variability implies cell function. _Cells_, 9(1), p. 14.

### Conference Publications
- Li, X., **Li, G.<sup>*</sup>**, and Zhang, X. (2025). A Likelihood-Based Approach for Watermark Detection. _International Conference on Artificial Intelligence and Statistics (AISTATS) 2025_.
- Li, X., **Li, G.<sup>*</sup>**, and Zhang, X. (2024). Segmenting Watermarked Texts From Language Models. _Conference on Neural Information Processing Systems (NeurIPS) 2024_.